packages/expo-updates/cli/src/utils/modifyConfigAsync.ts

```diff
@@ -19,21 +19,11 @@ export async function attemptModification(
   }
 }
 
-function logNoConfig() {
```

Comment:
I think the call hierarchy for this is only during an explicit execution of the expo-updates CLI, which should log when the CLI can't do what it needs to do (modify the config). I think we should revert this change?

Reply:
I updated the `modifyConfigAsync` function so it writes a config file when none exists, the only time we can't modify the config now is when it's a dynamic file (js/ts). We previously couldn't do this because parts of global CLI required a physical config file, but that's fully phased out now.

Reply:
Ah, got it.